.github/workflows/awstest.yml

```diff
@@ -25,10 +22,18 @@ jobs:
           run: conda install -c conda-forge awscli
         - name: Start AWS batch job
           env:
-            AWS_ACCESS_KEY_ID: ${{secrets.AWS_KEY_ID}}
-            AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_KEY_SECRET}}
-            TOWER_ACCESS_TOKEN: ${{secrets.TOWER_ACCESS_TOKEN}}
+            AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+            AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+            TOWER_ACCESS_TOKEN: ${{ secrets.AWS_TOWER_TOKEN }}
+            AWS_JOB_DEFINITION: ${{ secrets.AWS_JOB_DEFINITION }}
+            AWS_JOB_QUEUE: ${{ secrets.AWS_JOB_QUEUE }}
+            AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
           run: |
-            aws batch submit-job --region eu-west-1 --job-name nf-core-awstest --job-queue 'default-8b3836e0-5eda-11ea-96e5-0a2c3f6a2a32' --job-definition nextflow --container-overrides '{"command": ["nf-core/rnaseq", "-profile test --outdir s3://nf-core-awsmegatests/rnaseq/results -w s3://nf-core-awsmegatests/rnaseq/work -with-tower"],"environment": [{"name": "TOWER_ACCESS_TOKEN","value": "'"$TOWER_ACCESS_TOKEN"'"}]}'
+            aws batch submit-job \
+            --region eu-west-1 \
+            --job-name nf-core-awstest \
+            --job-queue $AWS_JOB_QUEUE \
+            --job-definition $AWS_JOB_DEFINITION \
+            --container-overrides '{"command": ["nf-core/rnaseq", "-r dev -profile test --outdir s3://'"${AWS_S3_BUCKET}"'/awsmegatests/results_'"${GITHUB_SHA}"' -w s3://'"${AWS_S3_BUCKET}"'/awsmegatests/work_'"${GITHUB_SHA}"' -with-tower"],"environment": [{"name": "TOWER_ACCESS_TOKEN","value": "'"$TOWER_ACCESS_TOKEN"'"}]}'
```

Comment:
Maybe we should add a note in the document, that the user needs to replace `rnaseq` with their pipeline name

Reply:
that's part of the pipeline template now, and it gets automatically replaced so it should be ok :)